package.json

```diff
@@ -12,7 +12,7 @@
     "test": "karma start"
   },
   "devDependencies": {
-    "@folio/stripes-core": "thefrontside/stripes-core#master",
+    "@folio/stripes-core": "2.7.0",
```

Comment:
Overlooked this. Same comment from the other PR: We'll need #master until they release the changes to the redux store setup

Reply:
Fixed.